packages/dd-trace/src/appsec/telemetry.js

```diff
@@ -0,0 +1,132 @@
+'use strict'
+
+const telemetryMetrics = require('../telemetry/metrics')
+
+const appsecMetrics = telemetryMetrics.manager.namespace('appsec')
+
+const DD_TELEMETRY_WAF_RESULT_TAG = Symbol('_dd.appsec.telemetry.waf.result.tag')
+
+const TAG = {
```

Comment:
I'm also used to write those with big caps but in this repo I've been asked to use normal camelCase. The props can be full caps tho no problem!
```suggestion
const tags = {
```